Library/Homebrew/dependable.rb

```diff
@@ -1,7 +1,7 @@
 require "options"
 
 module Dependable
-  RESERVED_TAGS = [:build, :optional, :recommended, :run, :linked]
+  RESERVED_TAGS = [:build, :optional, :recommended, :run]
```

Comment:
My concern is then we end up breaking random taps that have added it based on our `audit`.